services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationController.kt

```diff
@@ -101,21 +105,42 @@ class ApplicationController(
     fun create(@ValidApplication @RequestBody application: Application): Application {
         val userId = currentUserId()
         val hankeTunnus = application.hankeTunnus
-        val hankeId = hankeService.getHankeId(hankeTunnus)
 
-        if (
-            hankeId == null ||
-                !permissionService.hasPermission(hankeId, userId, PermissionCode.EDIT_APPLICATIONS)
-        ) {
-            throw HankeNotFoundException(hankeTunnus)
-        }
+        checkPermissionToCreate(hankeTunnus, userId)
 
         val createdApplication = service.create(application, userId)
 
         disclosureLogService.saveDisclosureLogsForApplication(createdApplication, userId)
         return createdApplication
     }
 
+    @PostMapping("/luo-hanke")
```

Comment:
Would this be clearer as "/johtoselvityshakemukset" without "/hakemukset" in front?

Somehow "/hakemukset/luo-hanke" feels cumbersome, and it breaks common conventions about RESTful URLs.

Reply:
Yes, that's fine.

Reply:
Fixed.